### PR TITLE
external-script: remove signature import

### PR DESCRIPTION
**Pull request by subhamkrai**
**Description of your changes:**

external-script: remove signature import
remove signature import as it is not required.

external-script: clean import section
import all the libraries at top

Signed-off-by: subhamkrai <srai@redhat.com>

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/development-flow.html#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by parth-gr — APPROVED** [review]
[external-rgw-quincy-test](https://github.com/rook/rook/runs/6653123964?check_suite_focus=true#logs) the CI looks green

File: deploy/examples/create-external-cluster-resources.py [hunk 1]
```diff
@@ -15,7 +15,6 @@
 '''
 
 import errno
-from inspect import signature
```
Comment (leseb) [inline deploy/examples/create-external-cluster-resources.py:18]: Is this removed because it's unused?

**Review by leseb — APPROVED** [review]

File: deploy/examples/create-external-cluster-resources.py [hunk 2]
```diff
@@ -970,10 +966,10 @@ def get_rgw_fsid(self):
         if self._arg_parser.rgw_skip_tls:
             verify = False
         base_url =  self.endpoint_dial(rgw_endpoint,cert=cert) + "://"
-        base_url = base_url + rgw_endpoint + "/admin/info?"   
+        base_url = base_url + rgw_endpoint + "/admin/info?"
         params = {'format': 'json'}
         request_url = base_url + urllib.parse.urlencode(params)
-            
+
```
Comment (leseb) [inline deploy/examples/create-external-cluster-resources.py:972]: @subhamkrai can we have a linter bot for this?

Comment (subhamkrai) [reply deploy/examples/create-external-cluster-resources.py:972]: right,  Ill take a look will open issue  to track

**Comment by subhamkrai** [comment]
@travisn we need to bp this to 1.9(missed to put bp level), I'll create a manual Bp after that we'll try to rebase #10408 might resolve the  conflicts if not I'll resolve manually

**Comment by subhamkrai** [comment]
@parth-gr fyi^^^

**Comment by subhamkrai** [comment]
> @travisn we need to bp this to 1.9(missed to put bp level), I'll create a manual Bp after that we'll try to rebase #10408 might resolve the conflicts if not I'll resolve manually

manual BP of this #10421

**Pull request merged**